pkg/controllers/restore/controller.go

```diff
@@ -369,6 +370,10 @@ func (h *handler) generateDependencyGraph(ownerToDependentsList map[string][]res
 		}
 
 		metadata := resourceData.Object[metadataMapKey].(map[string]interface{})
+		// ignore the secrets section in the service account to avoid missing token secret error in k8s 1.21
```

Comment:
I don't think it's k8s 1.21-specific. I've had this issue on local k3d cluster (`v1.19.8+k3s1`). To test, output any SA to yaml and then apply this yaml to another cluster. I bet it will have 2 secrets linked by it.

Reply:
the inline comment is updated.

Reply:
Ideally we want description of the PR updated too. :)

Reply:
done.